share/graphql/query.py

```diff
@@ -41,7 +41,7 @@ def resolve_me(self, args, context, info):
     @graphene.resolve_only_args
     def resolve_sources(self, limit=25, offset=0):
         # TODO Long title will need an index
-        return models.ShareUser.objects.exclude(long_title='').exclude(favicon='').order_by('long_title')[offset:offset + limit]
+        return models.ShareUser.objects.exclude(long_title='').exclude(icon='').order_by('long_title')[offset:offset + limit]
```

Comment:
See above comment about icon.

Does this result in an `AND` or an `OR` query/ What is it intended to do?